README.md

```diff
@@ -68,7 +68,7 @@ The above is semantically equivalent to the more explicit try-finally version:
 ```java
 io.opentracing.Tracer tracer = ...;
 ...
-ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive();
+ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive(new AutoFinisher());
```

Comment:
Opinion: I prefer no ActiveSpan type vs adding more options to it. This is generally discussed in #80 so not repeating here